### DISCUSSION TITLE
pyMOR Community Meeting 2024-01-09

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#2232] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, January 9, at 15:00 CET and 9:00 EST, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] future directions (@pmli)
- [x] new intern at MPI for the QR-related algorithms (@drittelhacker)
- [x] BBB meeting room (@drittelhacker)
- [x] github.com/pymor/docs (@sdrave)
- [ ] ...

Moderator: @pmli

**Comment by lbalicki** [comment]
Meeting Notes:

- github.com/pymor/docs
  - repo too large to be checked out on github
  - clean up process doesn't work as expected
  - suggestion: Move this to the ziv machines
  - downside: only a few people have access
  - agreed that this sounds good, will start working on moving things over
- new intern at MPI for the QR-related algorithms
  - work on block QR algorithms and benchmarks
- BBB meeting room
  - filed complaint about people dropping out of meeting
  - BBB is low priority, possibly we should consider alternative platform in the future
  - for now dropouts aren't too bad, so we can continue using BBB for now
- future directions
  - will schedule separate meeting do discuss future development

Next pyMOR Community Meeting January 23rd, 15:00 CET
